### ISSUE TITLE
[bug] There are 0 legends but undefined theme

### ISSUE THREAD

**Issue by at15** [dyweb/colorful-contribution#3]
I am using the Mario theme on Chrome 66.0.3359.139

in `content.js` line 51-55

````js
  // Check for the number of legends
  if (legends.length != theme.colors.length) {
    console.error('There are ' + legends.length + ' legends but ' + theme.length + ' theme')
    return
  }
````

**Comment by gigaflw** [comment]
working on this...

**Comment by gigaflw** [comment]
Does it happen constantly ?  If it does, do you have 5 icons like this
<img width="154" alt="screen shot 2018-05-02 at 11 54 35 am" src="https://user-images.githubusercontent.com/13916180/39504641-9ee3cd28-4dff-11e8-936d-03b3d94bc934.png"> displayed? @at15

**Comment by at15** [comment]
![screenshot from 2018-05-01 21-46-04](https://user-images.githubusercontent.com/5621298/39505783-93c30808-4d89-11e8-8134-9afc38f859e4.png)

I got five icons, but the order is not the same .... and there is no less and more

I am using the version on Chrome Web Store, I can try the master branch (If you have detailed instruction, I am lazy ...

**Comment by gigaflw** [comment]
Oh, I see. I haven't limited the script to the profile page only. Now it runs on every github page and give an error if there is no corresponding element.
This is hard to solve, however, since there is no obvious way to tell a profile page from other github pages from url. I can muffle the error info though.
Thanks for mentioning! 👍 @at15

**Comment by at15** [comment]
@gigaflw actually you can tell if a page is profile page

- first the url would be github.com/user name, i.e. https://github.com/at15
- second you need to deal with organization, which has the same url as user, there is no div for contribution on organization page https://github.com/dyweb

**Comment by gigaflw** [comment]
Yep, we still need to resort to DOM query to distinguish, instead of using urls only.
The extension is doing this now already (though implicitly). I'll make it explicit in the code, and remove the error message.

**Comment by gigaflw** [comment]
Fixed in v0.3.0
